starter_templates/rust/code/src/main.rs

```diff
@@ -20,6 +21,7 @@ fn main() {
         process::exit(1);
     }
 
+    #[allow(unused_variables)] // Feel free to remove this line once you uncomment the code below
```

Comment:
Hmm, this seems like we're adding stuff just to satisfy the linter. Doesn't actually fix genuine issues, our use-case is such that users do need to have unused code when starting out. @andy1li can we try disabling dead_code and unused_variables in the rust lint command?